app/konnect/mesh-manager/rbac.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Mesh Manager RBAC
+content_type: reference
+---
+
+Mesh Manager has it's own role-based access control (RBAC) settings that are separate from the {{site.konnect_short_name}} RBAC settings. The Mesh Manager RBAC settings are specific to the meshes and mesh policies in Mesh Manager. 
+
+To completely configure RBAC for Mesh Manager, you must configure both roles and role bindings.
+* **Role:** Determines what resources a user or group has access to
+* **Role binding:** Determines which users or groups are assigned to a particular role
+
+The Admin role and role binding is automatically created for you. The admin role can be used for the service mesh operators who are part of infrastructure team responsible for {{site.mesh_product_name}} deployment.
+
+## Key mapping
+
+### Roles
+
+Access roles specify access levels and resources to which access is granted. Access is only defined for write operations. Read access is available to all users who have the {{site.konnect_short_name}} Mesh global control plane `viewer` role. Access roles define roles that are assigned separately to users and groups/teams using access role bindings. They are scoped globally, which means they are not bound to a mesh. 
+
+For more information about how to configure the key mappings and RBAC settings, see [Role-Based Access Control](/mesh/latest/features/rbac/) in the {{site.mesh_product_name}} documentation.
+
+### Role binding
+
+| {{site.mesh_product_name}} key      | Description  |
+|-----------------------------|--------------|
+| `type` | The resource type. This will be `AccessRoleBinding`. |
```

Comment:
This line doesn't make sense to me. Is the type `accessrolebinding` by default? If so we can use the syntax default:`AccessRoleBinding`, but I do wonder what other types there are and if they are necessary to include.

Reply:
I revised it a bit, does this sound better? For role binding, it has to be `AccessRoleBinding`, but the same format is used for role mapping, so this technically could also be `AccessRole`, but then it would be for the MinK role and not the MinK role binding.

Reply:
yes